xml/obs_build_constraints.xml

```diff
@@ -188,6 +188,19 @@ Constraint: hardware:disk:size unit=M 4000
   <title>Constraint syntax</title>
   <para> This section describes the various constraint qualifiers and their
    syntax. </para>
+  <para> In general, it is important to understand that the syntax used will
+   differ depending on where the constraints are specified. When specified
+   via a <literal>_constraints</literal> file, XML syntax is used, while a
+   different syntax is used when specifying constraints in a project
+   configuration or a build recipe (e.g., an RPM spec file). In this section,
+   both syntaxes are described for each qualifier. </para>
+  <note><para>
+   When specifying constraints using XML syntax, attribute values must be
+   enclosed in double-quotes (<literal>unit="G"</literal>,
+   <literal>exclude="true"</literal>, etc.), while in project configurations and
+   build recipes the attributes must be given <emphasis>without</emphasis>
+   quotes (<literal>unit=G</literal>, <literal>exclude=true</literal>).
```

Comment:
This sounds like a bug that should be fixed instead of described to me...

Reply:
The follow-up question, I suppose, would be: "a bug in what?"

If we say it's a bug in the build constraints parser that it does not tolerate double-quotes around values (that cannot contain spaces), then we must also say it's a bug in XML that it requires double-quotes around values, even when those values do not include spaces?

Reply:
the parser is something we write, the XML standard not :)